crypto/crmf/crmf_pbm.c

```diff
@@ -148,7 +155,7 @@ int OSSL_CRMF_pbm_new(const OSSL_CRMF_PBMPARAMETER *pbmp,
      * compute the key used in the MAC process.  All implementations MUST
      * support SHA-1.
      */
-    if ((m = EVP_get_digestbyobj(pbmp->owf->algorithm)) == NULL) {
+    if ((owf = EVP_get_digestbyobj(pbmp->owf->algorithm)) == NULL) {
```

Comment:
I will have to look into what to do with this call...
See https://github.com/openssl/openssl/issues/11820

Reply:
I've been able to solve this; `test_cmp_server` now passes.